tools/base/fpgainfo/errors.c

```diff
@@ -180,7 +180,7 @@ static const char *const PORT_ERROR[PORT_ERROR_COUNT] = {
 /*
  * errors command configuration, set during parse_args()
  */
-static struct errors_config {
+STATIC struct errors_config {
```

Comment:
Is this needed after all?  Looks like the struct was re-defined inside the test file.